drivers/abp_pressure_sensor.py

```diff
@@ -1,46 +1,22 @@
-import pigpio
 import logging
 
-from errors import I2CReadError
-
-from .i2c_driver import I2cDriver
+from honeywell_pressure_sensor import HoneywellPressureSensor
```

Comment:
use relative import:
```suggestion
from .honeywell_pressure_sensor import HoneywellPressureSensor
```